geoportailv3/static/js/query/interrogationdirective.js

```diff
@@ -0,0 +1,234 @@
+goog.provide('app.queryDirective');
+
+
+/**
+ * @param {string} appQueryTemplateUrl URL to directive template.
+ * @return {angular.Directive} The Directive Definition Object.
+ * @ngInject
+ */
+app.queryDirective = function(appQueryTemplateUrl) {
+  return {
+    restrict: 'E',
+    scope: {
+      'map': '=appQueryMap',
+      'selectedLayers': '=appQuerySelectedlayers',
+      'infoOpen': '=appQueryOpen'
+    },
+    controller: 'AppQueryController',
+    controllerAs: 'ctrl',
+    bindToController: true,
+    templateUrl: appQueryTemplateUrl
+  };
+};
+app.module.directive('appQuery', app.queryDirective);
+
+
+
+/**
+ * @constructor
+ * @param {angular.Scope} $scope Scope.
+ * @param {angular.$http} $http Angular $http service
+ * @param {string} appQueryTemplatesPath Path
+ *                 to find the intterogation templates.
+ * @param {string} getInfoServiceUrl
+ * @param {string} getRemoteTemplateServiceUrl
+ * @export
+ * @ngInject
+ */
+app.QueryController = function($scope, $http,
+    appQueryTemplatesPath, getInfoServiceUrl,
+    getRemoteTemplateServiceUrl) {
+
+  this['content'] = [];
+  this.templatePath = appQueryTemplatesPath;
+  this.remoteTemplateUrl = getRemoteTemplateServiceUrl;
+  var map = this['map'];
+
+  var defaultFill = new ol.style.Fill({
+    color: [255, 255, 0, 0.6]
+  });
+  var circleStroke = new ol.style.Stroke({
+    color: [255, 155, 55, 1],
+    width: 3
+  });
+
+  var image = new ol.style.Circle({
+    radius: 10,
+    fill: defaultFill,
+    stroke: circleStroke
+  });
+
+  var defaultStyle = [
+    new ol.style.Style({
+      fill: new ol.style.Fill({
+        color: [255, 255, 0, 0.6]
+      })
+    }),
+    new ol.style.Style({
+      stroke: new ol.style.Stroke({
+        color: 'white',
+        width: 5
+      })
+    }),
+    new ol.style.Style({
+      stroke: new ol.style.Stroke({
+        color: '#ffcc33',
+        width: 3
+      })
+    })
+  ];
+
+  var styles = {
+    'Point': [new ol.style.Style({
+      image: image
+    })],
+    'default': defaultStyle
```

Comment:
If your Angular templates do not include references to that property you do need to use quotes. Quotes prevent renaming, they should be used only when necessary.